exif-utils.c:88:
    error: static declaration of 'exif_get_sshort' follows non-static declaration
../libexif/exif-utils.h:48:
    error: previous declaration of 'exif_get_sshort' was here

--- libexif-0.6.12/libexif/exif-utils.c.orig	2005-03-13 03:27:13.000000000 +0100
+++ libexif-0.6.12/libexif/exif-utils.c	2005-03-17 01:19:01.000000000 +0100
@@ -83,7 +83,7 @@
 	}
 }
 
-static ExifSShort
+ExifSShort
 exif_get_sshort (const unsigned char *buf, ExifByteOrder order)
 {
 	if (!buf) return 0;
